workflow-templates/deploy-preview.yml

```diff
@@ -0,0 +1,67 @@
+on:
+  pull_request:
+    branches: [ $default-branch ]
```

Comment:
`$default-branch` picks up whatever default branch the repo has set

Reply:
TIL, good to know